featurebyte/service/feature_materialize.py

```diff
@@ -288,7 +288,7 @@ async def initialize_new_columns(
         materialize_partial(
             feature_store=feature_store,
             feature_view=feature_store.get_feature_view(feature_table_model.name),
-            columns=materialized_features.column_names,
+            columns=materialized_features.column_names + [InternalName.FEATURE_TIMESTAMP_COLUMN],
```

Comment:
@YS-L Currently, the ingest feature timestamp is not in the online feature store. Adding an ingest timestamp column here to populate the timestamp column to feature store. Please let me know if there are better way to handle this.

Reply:
This is done only for `initialize_new_columns()`. Should `scheduled_materialize_features()` also be updated? Should we check whether a feature table require ttl handling before adding this timestamp column?

Reply:
> This is done only for `initialize_new_columns()`. Should `scheduled_materialize_features()` also be updated?
Not sure the reason to update `scheduled_materialize_features`. Would be good if you could clarify this. The main purpose of this update is to populate the online stores with table's ingested timestamp.

```
Should we check whether a feature table require ttl handling before adding this timestamp column?
```
It would be better if we could optimize this part.

Reply:
Is the ingested timestamp never updated after being populated? I thought it should be updated every time the feature is computed (done regularly by `scheduled_materialize_features()`), since it refers to the timestamp at which the feature is computed.

Reply:
Ok. I think i got it. Do you mean we should also update `materialize_partial` in `scheduled_materialize_features`?

Reply:
Yes, that's what I meant - otherwise the ingested timestamp in the online store will always be the initial value.